db/db_test.cc

```diff
@@ -5101,7 +5103,7 @@ TEST_F(DBTest, AutomaticConflictsWithManualCompaction) {
       [&](void* arg) { callback_count.fetch_add(1); });
   rocksdb::SyncPoint::GetInstance()->EnableProcessing();
   CompactRangeOptions croptions;
-  croptions.exclusive_manual_compaction = false;
+  croptions.exclusive_manual_compaction = true;
```

Comment:
interesting! This test failed on the blame PR, but with the following error message so I ignored it :(

[ RUN      ] DBTest.AutomaticConflictsWithManualCompaction
db/db_test.cc:5103: Failure
db_->CompactRange(croptions, nullptr, nullptr)
IO error: No space left on deviceWhile appending to file: /tmp/rocksdbtest-2000/db_test/000553.sst: No space left on device

Reply:
totally logical to ignore 😂